packages/realm-react/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 ## vNext (TBD)
 
 ### Enhancements
+* Add authentication hooks, `useAuth` and `useEmailPasswordAuth`
+	Usage example:
+	```tsx
```

Comment:
Gentle reminder to add an example

Reply:
I think i'll just link to the README section, as adding usage examples here would be a bit overwhelming